src/Config/SettingsTab.ts

```diff
@@ -125,6 +125,17 @@ export class SettingsTab extends PluginSettingTab {
                 });
             });
 
+        new Setting(containerEl)
+            .setName('Set created date on every added task')
+            .setDesc('Enabling this will add a timestamp ➕ YYYY-MM-DD at the end when a task is created')
```

Comment:
Please could this say "... when a task is created **with 'Create or edit task'**".

Otherwise I will be deluged with support requests saying they typed a new task and the date was not added. 😆

Reply:
Add 'Create or edit task'.